pkg/engine/jmespath/arithmetic_test.go

```diff
@@ -768,6 +768,106 @@ func Test_Modulo(t *testing.T) {
 	}
 }
 
+func Test_Round(t *testing.T) {
+	testCases := []struct {
+		name           string
+		test           string
+		expectedResult interface{}
+		err            bool
+		retFloat       bool
+	}{
+		// Quantity
+		{
+			name: "Quantity roundoff Quantity -> error",
+			test: "round('31Ki', '13Ki')",
+			err:  true,
+		},
+		{
+			name: "Quantity roundoff Duration -> error",
+			test: "round('12Ki', '13s')",
+			err:  true,
+		},
+		{
+			name:           "Quantity roundoff Scalar -> Quantity",
+			test:           "round('12Ki', `2`)",
```

Comment:
What does that mean to round a quantity ?

Reply:
Quantity can never be in a floating point number, So IG there is no meaning to round of quantity, so it returns the same quantity as entered.

Reply:
Shall it be considered an error to call round on a quantity ?